README.md

```diff
@@ -49,8 +65,32 @@ You must to issue personal access token in advance.
 $ COMPOSER_PROCESS_TIMEOUT=0 composer run
 ```
 
+If you use Docker, alternatively run below command.
```

Comment:
"below" is a prepositionl, use "following command".

Reply:
fixed https://github.com/standfirm/esaba/pull/2/commits/b3851fb9dbf3acbb08caade0c50302f22c222880